packages/dara-components/js/smart/chat/chat.tsx

```diff
@@ -149,60 +187,64 @@ function Chat(props: ChatProps): JSX.Element {
+        setAreThereOtherChats(checkMoreThanOneRenderedElement('.chatThread'));
+    };
+
+    React.useLayoutEffect(() => {
+        setAreThereOtherChats(checkMoreThanOneRenderedElement('.chatThread'));
+    }, [showChat]);
+
     return (
-        <ChatWrapper>
+        <>
             {showChat && (
-                <StyledChat
-                    $rawCss={css}
-                    className={props.className}
-                    onClose={() => setShowChat(false)}
-                    onUpdate={onUpdate}
-                    style={style}
-                    value={value}
-                    activeUser={parseUserData(userData)}
-                />
-            )}
-            {!showChat && (
-                <ChatButton onClick={() => setShowChat(true)}>
-                    <svg fill="none" height="32" viewBox="0 0 32 32" width="32" xmlns="http://www.w3.org/2000/svg">
-                        <rect fill="none" height="24" rx="3" width="30" x="1" y="1.33594" />
-                        <rect
-                            height="24"
-                            rx="3"
-                            stroke={theme.colors.background}
-                            strokeWidth="2"
-                            width="30"
-                            x="1"
-                            y="1.33594"
-                        />
-                        <path
-                            d="M8 8.33594H24"
-                            stroke={theme.colors.background}
-                            strokeLinecap="round"
-                            strokeWidth="2"
-                        />
-                        <path
-                            d="M8 13.3359H24"
-                            stroke={theme.colors.background}
-                            strokeLinecap="round"
-                            strokeWidth="2"
-                        />
-                        <path
-                            d="M8 18.3359H24"
-                            stroke={theme.colors.background}
-                            strokeLinecap="round"
-                            strokeWidth="2"
-                        />
-                        <path
-                            d="M18.5981 26.1641L16 30.6641L13.4019 26.1641L18.5981 26.1641Z"
-                            fill={theme.colors.background}
-                            stroke={theme.colors.background}
-                        />
-                        <path d="M16 28.3359L13.4019 23.8359L18.5981 23.8359L16 28.3359Z" fill="none" />
-                    </svg>
-                </ChatButton>
+                // we set the z-index so that the latest chat thread opened is always on top, and if there is a chat thread open, we set the background color so that the transparency does not show the thread behind
+                <ThreadWrapper
+                    className="chatThread"
```

Comment:
class names should be kebab-case

Reply:
Oops had it as an id before and forgot to change conventions, fixed :)